projects/Forgejo/default.nix

```
@@ -0,0 +1,15 @@
+{
+  pkgs,
+  lib,
+  sources,
+} @ args: let
+  inherit (lib) modulePath;
+in {
+  packages = {
+    inherit (pkgs) forgejo;
+  };
+
+  nixos = {
+    modules.services.forgejo = "${modulePath}/services/misc/forgejo.nix";
```

If we're taking this route, why can't we say more directly
 
```suggestion
    modules.services.forgejo = "${sources.nixpkgs}/nixos/modules/services/misc/forgejo.nix";
```

But I'm not sure whether hard-coding the paths is a good idea. Those may change arbitrarily, as they're not part of the external interface. Don't have to do it now, but it seems more explicit and non-magic to be able to say something like

```nix
modules.services.forejo = nixos.modules.services.forejo;
```

where we get `nixos` as an argument from somewhere

---

I currently don't have a clear mental model of how to implement the following

> ```nix
> modules.services.forejo = nixos.modules.services.forejo;
> ```

I'll go with `"${sources.nixpkgs}/nixos/modules/services/misc/forgejo.nix";` for now